zigzag/zigzag_test_log.py

```diff
@@ -599,16 +635,126 @@ def __init__(self, testcase_name, teststeps_xml, mediator):
                 a single qTest test log.
             mediator (ZigZag): The mediator that stores shared data.
         """
-
-        # Initialize without parsing so we can post-process the test steps.
-        super(_ZigZagTestLogWithSteps, self).__init__(None, mediator, auto_parse=False)
+        super(_ZigZagTestLogWithSteps, self).__init__(None, mediator)
 
         self._name = testcase_name
         self._teststeps_xml = teststeps_xml
         self._zz_test_step_logs = []
         self._qtest_test_step_logs = []
 
-        self._parse()
+        # Convert the test steps into test logs for easier post-processing.
+        self._zz_test_step_logs = \
+            [_ZigZagTestLog(ts_xml, self._mediator) for ts_xml in self._teststeps_xml]
+
+    @property
+    def errors(self):
+        """Gets the errors form the testcase_xml
+
+        Returns:
+            list: the list of errors found
+        """
+        if self._errors is None:
+            self._errors = [error for log in self._zz_test_step_logs for error in log.errors]
```

Comment:
Oh look at that! You figured it out!

![Bravo](https://media.giphy.com/media/l4q7TIW8nEZYOJUf6/giphy-downsized.gif)